files/en-us/web/css/border/index.html

```diff
@@ -91,7 +91,7 @@ <h3 id="Setting_a_pink_outset_border">Setting a pink outset border</h3>
 
 <h4 id="HTML">HTML</h4>
 
-<pre class="brush: html notranslate">&lt;div&gt;I have a border, an outline, AND a box shadow! Amazing, isn't it?&lt;/div&gt;
+<pre class="brush: html notranslate">&lt;div&gt;I have a border, an outline and a box shadow! Amazing, isn't it?&lt;/div&gt;
```

Comment:
```suggestion
<pre class="brush: html notranslate">&lt;div&gt;I have a border, an outline, and a box shadow! Amazing, isn't it?&lt;/div&gt;
```

Oxford comma clarifies the `div`'s three properties.

Reply:
I could commit but really?
Can You please link something?

Reply:
https://www.nytimes.com/2018/02/09/us/oxford-comma-maine.html